manage_arkime/core/capacity_planning.py

```diff
@@ -0,0 +1,93 @@
+from dataclasses import dataclass
+import math
+import logging
+
+logger = logging.getLogger(__name__)
+
+INSTANCE_TYPE_CAPTURE_NODE = "m5.xlarge" # Arbitrarily chosen
+TRAFFIC_PER_M5_XL = 2 # in Gbps, guestimate, should be updated with experimental data
```

Comment:
Should this just be TRAFFIC_PER_CAPTURE_NODE, why hard code the node type in the variable names?

Reply:
Nothing huge, just how I ended up coding it.  The traffic amount listed is tied to the specific instance type.  We may want to change the instance type later.  If we do, we may want to keep the old, known-good mapping between m5.xl and 2 gbps in a constant in the code and just add a new constant mapping between traffic and the new instance type.

Reply:
My thoughts is, its fine if you want a table or whatever at the top with these values, but the code itself should probably use a generic variable or something instead of using the variable with node type hard coded. This can wait until we actually change if easier. :)